tests/instancetype/instancetype.go

```diff
@@ -711,6 +723,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		createAndValidateVirtualMachine := func() {
 			By("Creating the VirtualMachine")
 			var err error
+			libvmi.WithRunStrategy(virtv1.RunStrategyAlways)(vm)
```

Comment:
This test suite is making heavy use of vm.spec.running so might collide with runstrategy

Reply:
https://github.com/kubevirt/kubevirt/blob/cb9822a51211c52cb933eead253919077f159bd7/tests/instancetype/instancetype.go#L832

Reply:
`libvmi.WithRunStrategy` set `running` to nil

Reply:
@akalenyu 
You are right, `libvmi.WithRunStrategy` doesn't set running to `nil`. I was confused. Thanks for the note.
It should work now.

Reply:
The issue here is exactly that `libvmi.WithRunStrategy` does not set `running` to nil.
I remember that it was requested. Anyway, I think it is worth it to fix the `libvmi.WithRunStrategy` function at all, also to align the behavior with `WithRunning()` (which sets the `runStrategy` to nil)

Reply:
@EdDev (libvmi owner) was adamantly against complicating `WithRunStrategy`. He believes that `libvmi` should not be in the business of glossing over problems in KubeVirt API, and would like `WithRunning` to align.

Reply:
Yep, which makes total sense. The real issue is that this suite is now mixing between running and runstrategy

Reply:
That's fine. I think the "complication" would have easily disappeared alongside the deprecation and removal of the `Running` field (https://github.com/kubevirt/kubevirt/issues/11993).
Just my 2 cents.